### ISSUE TITLE
Link to analysis does not display results

### ISSUE THREAD

**Issue by matthewy98** [veg/hyphy#1320]
Hi,

I have ran several analysis this evening on nucleotide sequences using both MEME and SLAC methods, however only one link has provided any results that open, and the others just show a blank screen with the twitter mentions displayed heres one for example, https://www.datamonkey.org/slac/6074c298d35f05276b49a62b 

Can anyone help with this?

Thank you!

**Comment by stevenweaver** [comment]
Dear @matthewy98,

Thank you for pointing out this issue. The issue stems from hyphy outputting invalid JSON. Specifically, `inf` is used to represent infinity, when using a high value, such as `1e500`, is the only valid way to represent infinity as a numerical value. I've adjusted the results json for the linked job, so it should now render. As far as addressing this issue in its entirety, it will require either post-processing on the behalf of datamonkey to remove `inf`, or preferably updating `hyphy` itself. 

Keeping this issue open for now until a decision is made regarding remediation. 

Best,
Steven

**Comment by matthewy98** [comment]
Hi Stephen

Thanks for your reply,

I don’t know how to post-process to remove inf, or update hyphy itself, do you have any suggestions on how I can continue running SLAC analysis of fasta files of nucleotide sequences?

Best wishes,

Matthew

**Comment by spond** [comment]
Dear @matthewy98,

I'll take a look at the underlying dataset and see what the issue is. Will attempt to push a fix to 2.5.31 which will then deploy to Datamonkey

Best,
Sergei

**Comment by spond** [comment]
Dear @matthewy98,

The current version of HyPhy identifies an issue with the data file:

```
### Obtaining branch lengths and nucleotide substitution biases under the nucleotide GTR model
Error:
'E' found at sites {3819} of sequence 'EPI_ISL_940614_BRAZIL_AM_1059_2021_2021_01_19' maps to a null probability vector; this will result in an uncomputable likelihood function
```

Some of your sequences appear to have the letter `E` at the end e.g.

```
GAAGGGCTGCTGCAGCTGCGGCAGCTGCTGCAAGTTCGACGAGGACGACAGCGAGCCCGTGCTGAAGGGCGTGAAGCTGCACTACACCE--
```

`E` is not an allowed nucleotide, so HyPhy now rejects these files. In earlier versions (including the one running SLAC on Datamonkey), `E` is allowed by results in an `-Inf` likelihood, which triggered the issues you originally reported.

Would you mind checking your input sequences for the presence of "stray" nucleotides?

Best,
Sergei
